RCS file: /cvs/util/officecfg/data/org/openoffice/Office/Common.xcd,v
retrieving revision 1.83
diff -u -r1.83 Common.xcd
--- officecfg/data/org/openoffice/Office/Common.xcd
+++ officecfg/data/org/openoffice/Office/Common.xcd
@@ -3644,7 +3644,7 @@
 					</schema:value-names>
 					<schema:constraints xmlns:xsd="http://www.w3.org/1999/XMLSchema"/>
 				</schema:type-info>
-				<default:data>0</default:data>
+				<default:data>1</default:data>
 			</schema:value>
 			<schema:value cfg:name="ToolboxStyle" cfg:type="short">
 				<!-- Responsible: MBA -->
